{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5cb42c87-64d0-42d0-94cd-e4e4fe97e2bb",
   "metadata": {},
   "outputs": [],
   "source": [
    "#!/bin/bash \n",
    "#SBATCH -J HTSeqcount_batch\n",
    "#SBATCH -A naiss2023-5-472\n",
    "#SBATCH -p core \n",
    "#SBATCH -n 1\n",
    "#SBATCH -t 06:00:00 \n",
    "\n",
    "module load bioinfo-tools\n",
    "module load htseq/2.0.2\n",
    "\n",
    "L = $1\n",
    "\n",
    "htseq-count \\\n",
    "--format=bam \\\n",
    "-r pos \\\n",
    "--stranded=reverse \\\n",
    "-c csv \\\n",
    "/proj/sexsel_data/snic2020-16-129_whole_project_directory/private/HS_MR_remake/samtools/filtered/$1_filt.BAM \\\n",
    "/proj/sexsel_data/snic2020-16-129_whole_project_directory/private/Stockholm_RNA_delivery06265/VD-3257/alex/newgenome/new_anno.gtf \\\n",
    "> $1_counts.txt\n",
    "\n",
    "# i think you can specify the output file with -c /OR/ > but not both maybe?\n",
    "\n",
    "\n",
    "\n",
    "OUTPUT ERROR \n",
    "\n",
    "  Format not recognized for output count file: aff_s42_reverse\n",
    "  [Exception type: ValueError, raised in utils.py:371]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "02b52417-7fa7-4153-a63f-aa5ff08b959e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# MR batches\n",
    "\n",
    "\n",
    "#!/bin/bash\n",
    "#SBATCH -J MR_Batch_1\n",
    "#SBATCH -A naiss2023-5-472\n",
    "\n",
    "for sample in AV3_S55 AV2_S54 AV1_S53 ALH_S52 ALG_S51 ALF_S50 ALE_S49 ALD_S48 ALC_S47 ALB_S46 ALA_S45 AFH_S44 AFG_S43 AFF_S42\n",
    "do\n",
    "    sbatch HTSeqCount_batch.sh \"$sample\"\n",
    "done\n",
    "\n",
    "\n",
    "\n",
    "#!/bin/bash\n",
    "#SBATCH -J MR_Batch_2\n",
    "#SBATCH -A naiss2023-5-472\n",
    "\n",
    "for sample in AFE_S41 AFD_S40 AFC_S39 AFB_S38 AFA_S37 AMH_S36 AMG_S35 AMF_S34 AME_S33 AMD_S32 AMC_S31 AMB_S30 AMA_S29\n",
    "do\n",
    "    sbatch HTSeqCount_batch.sh \"$sample\"\n",
    "done\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "06bb964e-8272-470d-a362-f6e63c0337c3",
   "metadata": {},
   "outputs": [],
   "source": [
    "# HS batches\n",
    "\n",
    "#!/bin/bash\n",
    "#SBATCH -J HS_Batch_1\n",
    "#SBATCH -A naiss2023-5-472\n",
    "\n",
    "for sample in FM1L1 FM1L2 FM1M1 FM1M2 FM1S1 FM1S2 FM2L4 FM2M1 FM2M4 FM2S1 FM2S2 FM3L1 FM3L7 FM3M1 FM3S1\n",
    "do\n",
    "    sbatch HTSeqCount_batch.sh \"$sample\"\n",
    "done\n",
    "\n",
    "\n",
    "\n",
    "#!/bin/bash\n",
    "#SBATCH -J HS_Batch_2\n",
    "#SBATCH -A naiss2023-5-472\n",
    "\n",
    "for sample in FST1L1 FST1L2 FST1L5 FST1M1 FST1M2 FST1S1 FST1S2 FST1S5 FST2L4 FST2L7 FST2M1 FST2M2 FST2S1 FST2S2 FST3L1 FST3L7 FST3M1 FST3M2 FST3M5 FST3S1 FST3S2\n",
    "do\n",
    "    sbatch HTSeqCount_batch.sh \"$sample\"\n",
    "done\n",
    "\n",
    "\n",
    "\n",
    "#!/bin/bash\n",
    "#SBATCH -J HS_Batch_3\n",
    "#SBATCH -A naiss2023-5-472\n",
    "\n",
    "for sample in HS1L1 HS1L2 HS1L7 HS1M1 HS1M2 HS1S1 HS1S2 HS2L4 HS2M1 HS2S1 HS2S2 HS3L1 HS3M1 HS3M2 HS3S1\n",
    "do\n",
    "    sbatch HTSeqCount_batch.sh \"$sample\"\n",
    "done\n",
    "\n",
    "\n",
    "\n",
    "#!/bin/bash\n",
    "#SBATCH -J HS_Batch_4\n",
    "#SBATCH -A naiss2023-5-472\n",
    "\n",
    "for sample in IRR1L1 IRR1M1 IRR1S1 IRR1S2 IRR2L4 IRR2L7 IRR2M1 IRR2M2 IRR2S1 IRR2S2 IRR3L1 IRR3L4 IRR3M1 IRR3M2\n",
    "do\n",
    "    sbatch HTSeqCount_batch.sh \"$sample\"\n",
    "done\n",
    "\n",
    "\n",
    "\n",
    "#!/bin/bash\n",
    "#SBATCH -J HS_Batch_4\n",
    "#SBATCH -A naiss2023-5-472\n",
    "\n",
    "for sample in IRR3S1 TRD1L1 TRD1M1 TRD1S1 TRD2L4 TRD2L7 TRD2M1 TRD2M2 TRD2S1 TRD2S2 TRD3L1 TRD3L7 TRD3M1 TRD3M2 TRD3S1 TRD3S2\n",
    "do\n",
    "    sbatch HTSeqCount_batch.sh \"$sample\"\n",
    "done\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "",
   "name": ""
  },
  "language_info": {
   "name": ""
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
